actions/upload-repository/action.yml

```diff
@@ -14,11 +14,17 @@ inputs:
     description: 'relative published artifact path (only useful with gh_pages)'
     required: false
     default: "targets"
+  ref:
+    description: 'Ref to clone'
+    required: false
+    default: ''
```

Comment:
this is related to my question in tuf-on-ci-template:
* what does '' mean as checkout ref?
* should this actually be `required: true` since we never use it without a ref arg

Reply:
The default is the empty string, which is forwarded to `actions/checkout`:
```
    # The branch, tag or SHA to checkout. When checking out the repository that
    # triggered a workflow, this defaults to the reference or SHA for that event.
    # Otherwise, uses the default branch.
    ref: ''
```

So if this is not provided, it will work as it did before, i.e it pulls the "latest" commit. I wrote latest in "" as there is no guarantees that you will actually get the commit pushed in the signing job.

Reply:
pedantically I don't think the default ever gets used though: the empty value comes from the caller.

Anyway, if it works that's good enough for me :+1: